content/about/people/SanjeFenkart/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Sanje Fenkart"
+description: "Sanje's presentation"
+draft: false
+
+type: "people"
+
+params:
+    wp1: true
+    institute: CERN
+    media:
+        email: sanje.antona.fenkart@cern.ch
+        github: SanjeAntona
+        linkedin: sanje-fenkart-656823255?l
+---
+{{< figure src="feature.jpg" alt= "photo of Sanje, (C) Christin Hristova, Christop Markus Rist" class="float-right m-2 w-[30%]" >}} 
+
+Sanje works as a Communications officer for the EVERSE project. She came to CERN in January 2023, where she was first part of the editorial team of the [CERN Courier](https://cerncourier.com), a bi-monthly magazin reporting on particle physics worlwide. Currently, she promotes EVERSE through newsletters, social media and webpage updates.
+
+Before coming to CERN, Sanje graduated in astrophysics at the [University of Vienna](https://astro.univie.ac.at/en/). During her studies she worked as a science communicator at the planetaria of Vienna and Klagenfurt, and joined a volunteer project  on journalism - [Wolfgang Magazin](https://www.wolfgang-magazin.com/author/sanje-fenkart/) - where she joined the science editorial team.
```

Comment:
- planetariums
- change the second "joined" to "became a member of"

Reply:
done